src/ErrorBoundary.jsx

```diff
@@ -37,96 +38,90 @@ class ErrorBoundary extends React.Component {
               }}
-            >
-              <Button
-                style={{
-                  width: '100%',
-                  marginTop: 12,
-                }}
-                display="primary"
-                onClick={() => {
-                  window.location.reload();
-                }}
-                id="RELOAD_THE_PAGE"
-              />
-              <ButtonLink
-                style={{
-                  width: '100%',
-                  marginTop: 8,
-                }}
-                display="secondary"
-                onClick={() => this.setState(initialState)}
-                href="/"
-                id="RETURN_HOME"
-              />
-              <ButtonLink
-                style={{
-                  width: '100%',
-                  marginTop: 8,
-                }}
-                href="https://community.wildme.org/"
-                display="tertiary"
-                external
-                newTab
-                id="REPORT_TO_COMMUNITY"
-              />
-            </div>
-            <div
-              style={{ marginTop: 20, width: '100%', maxWidth: 400 }}
-            >
-              <Accordion>
-                <AccordionSummary expandIcon={<ExpandMoreIcon />}>
-                  <Text variant="h6" id="VIEW_ERROR_DETAILS" />
-                </AccordionSummary>
-                <AccordionDetails
-                  style={{ display: 'flex', flexDirection: 'column' }}
-                >
-                  <Text variant="h5">{this.state.errorName}</Text>
-                  <Text variant="body2">
-                    {this.state.errorMessage}
-                  </Text>
-                </AccordionDetails>
-              </Accordion>
-            </div>
-          </Paper>
+              display="secondary"
+              onClick={() => this.setState(initialState)}
+              href="/"
+              id="RETURN_HOME"
+            />
+            <ButtonLink
+              style={{
+                width: '100%',
+                marginTop: 8,
+              }}
+              href="https://community.wildme.org/"
+              display="tertiary"
+              external
+              newTab
+              id="REPORT_TO_COMMUNITY"
+            />
+          </div>
+          <div
+            style={{ marginTop: 20, width: '100%', maxWidth: 400 }}
+          >
+            <Accordion>
+              <AccordionSummary expandIcon={<ExpandMoreIcon />}>
+                <Text variant="h6" id="VIEW_ERROR_DETAILS" />
+              </AccordionSummary>
+              <AccordionDetails
+                style={{ display: 'flex', flexDirection: 'column' }}
+              >
+                <Text variant="h5">{this.state.errorName}</Text>
+                <Text variant="body2">{this.state.errorMessage}</Text>
+              </AccordionDetails>
+            </Accordion>
+          </div>
         </div>
       </div>
     );
   }
 }
 
-export default ErrorBoundary;
+export default withTheme(ErrorBoundary);
```

Comment:
Had to look this up. Cool/weird.

Reply:
yeah these class components are the "old way" react used to work. however for some reason it appears error boundaries are still supposed to use them? hence the difference for this one component